src/main/mainEvents.ts

```diff
@@ -43,6 +44,18 @@ interface MainEventNames {
    */
   'settings-write'(settings: RecursivePartial<Settings>): void;
 
+  /**
```

Comment:
It looks like there's an extra indented-space here. `$ npm run lint --fix` should clean it up real quick 🙂

Reply:
Fixed! Thanks.